ehrapy/api/anndata_ext.py

```diff
@@ -32,13 +33,18 @@ def df_to_anndata(
     # if data is numerical only, short-circuit AnnData creation to have float dtype instead of object
     all_num = all(np.issubdtype(column_dtype, np.number) for column_dtype in dataframes.df.dtypes)
     X = dataframes.df.to_numpy(copy=True)
-
+    # initializing an OrderedDict with a non-empty dict might not be intended,
+    # see: https://stackoverflow.com/questions/25480089/right-way-to-initialize-an-ordereddict-using-its-constructor-such-that-it-retain/25480206
+    uns = OrderedDict()
+    # store all numerical columns (used for normalization)
+    uns["numerical_columns"] = list(df.select_dtypes("number").columns)
```

Comment:
Should we also add a "categorical" columns here which are all columns except these? A simple diff?

I am sure that at some point this will be useful.

Reply:
Might be useful, could also reuse this then in `autodetect`, for example